src/main/scala/com/sandinh/paho/akka/MqttPubSub.scala

```diff
@@ -143,7 +148,7 @@ import MqttPubSub._
   * 1. MqttClientPersistence will be set to null. @see org.eclipse.paho.client.mqttv3.MqttMessage#setQos(int)
   * 2. MQTT client will auto-reconnect
   */
-class MqttPubSub(cfg: PSConfig) extends FSM[S, Unit] {
+class MqttPubSub(cfg: PSConfig, connectListener: Option[ActorRef] = None) extends FSM[S, Unit] {
```

Comment:
Are `connectListener` param and `object ConnectedToBroker` not related to issue #2?